Test/Program.cs

```diff
@@ -117,9 +119,47 @@ private static void PrintFileInfo(string file, string outputDirectory, Options o
 
                 // Try to find the overlay header
                 if (!WiseOverlayHeader.FindOverlayHeader(stream, options.Debug, out var header) || header == null)
-                {
-                    _statistics.AddInvalidPath(file);
-                    Console.WriteLine($"No valid header could be found in {file}, skipping...");
+                { 
```

Comment:
Can you please put this path into a separate method? It will make it a bit easier to integrate into the Serialization changes later.